src/views/CrontabForm/CrontabForm.tsx

```diff
@@ -0,0 +1,236 @@
+import { CRONTAB_APIGROUP, CRONTAB_APIVERSION, CRONTAB_KIND } from "src/const";
+import {
+  useK8sModel,
+  k8sCreate,
+  useActiveNamespace,
+} from "@openshift-console/dynamic-plugin-sdk";
+import { useNavigate } from "react-router-dom-v5-compat";
+import { useCronTabTranslation } from "@crontab-utils/hooks/useCronTabTranslation";
+
+export const CronTabForm: React.FC = () => {
+  const [model] = useK8sModel({
+    group: CRONTAB_APIGROUP,
+    version: CRONTAB_APIVERSION,
+    kind: CRONTAB_KIND,
+  });
+  const [name, setName] = useState("");
+  const [cronSpec, setCronSpec] = useState("");
+  const [image, setImage] = useState("");
+  const [replicas, setReplicas] = useState<number | "">(1);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const navigate = useNavigate();
+  const { t } = useCronTabTranslation();
+  const [activeNamespace] = useActiveNamespace();
+  const minValue = 1;
+  const maxValue = 10;
+
+  const normalizeBetween = (value: number, min: number, max: number) => {
+    if (min !== undefined && max !== undefined) {
+      return Math.max(Math.min(value, max), min);
+    }
+    return value;
+  };
+
+  const onReplicasChange = (event: React.FormEvent<HTMLInputElement>) => {
+    const newValue =
+      event.currentTarget.value === ""
+        ? ""
+        : parseInt(event.currentTarget.value, 10);
+    setReplicas(newValue);
+  };
+
+  const onReplicasMinus = () => {
+    const newValue = normalizeBetween(
+      (replicas as number) - 1,
+      minValue,
+      maxValue
+    );
+    setReplicas(newValue);
+  };
+
+  const onReplicasPlus = () => {
+    const newValue = normalizeBetween(
+      (replicas as number) + 1,
+      minValue,
+      maxValue
+    );
+    setReplicas(newValue);
+  };
+
+  const onReplicasBlur = (event: React.FocusEvent<HTMLInputElement>) => {
+    const blurVal = +event.target.value;
+
+    if (blurVal < minValue) {
+      setReplicas(minValue);
+    } else if (blurVal > maxValue) {
+      setReplicas(maxValue);
+    } else if (isNaN(blurVal)) {
+      setReplicas("");
+    }
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    setLoading(true);
+    setError("");
+
+    const data = {
+      apiVersion: CRONTAB_APIVERSION,
+      kind: CRONTAB_KIND,
+      metadata: {
+        name,
+        namespace: activeNamespace,
+      },
+      spec: {
+        cronSpec,
+        image,
+        replicas: parseInt(replicas as string, 10),
+      },
+    };
+
+    k8sCreate({ model, data })
+      .then(() => {
+        setLoading(false);
+        useActiveNamespace;
+        navigate(`/k8s/ns/${activeNamespace}/stable.example.com~v1~CronTab`);
+      })
+      .catch((err) => {
+        setLoading(false);
+        setError(`Error creating CronTab: ${err.message}`);
+      });
+  };
+  return (
+    <>
+      <PageSection>
+        <Title headingLevel="h1">Create Form</Title>
+        <Form onSubmit={handleSubmit}>
```

Comment:
We should either have the handler on the button, with type="button", or on the Form, with button type="submit". They should be mutually exclusive to prevent confusion and hidden bugs.
```suggestion
        <Form>
```